plugins/plugin-docker/che-plugin-openshift-client/src/main/java/org/eclipse/che/plugin/openshift/client/OpenShiftConnector.java

```diff
@@ -178,7 +178,7 @@ public OpenShiftConnector(DockerConnectorConfiguration connectorConfiguration,
                               DockerConnectionFactory connectionFactory,
                               DockerRegistryAuthResolver authResolver,
                               DockerApiVersionPathPrefixProvider dockerApiVersionPathPrefixProvider,
-                              @Named("che.docker.ip.external") String cheServerExternalAddress,
+                              @Nullable @Named("che.docker.ip.external") String cheServerExternalAddress,
```

Comment:
It has to be `Nullable` to match the `ServerEvaluationStrategies`. Otherwise we break functionality on the Docker side, since this class is injected at bootstrap.